Gemfile

```diff
@@ -24,68 +21,69 @@ end
+gem "manageiq-messaging",               "~>1.0",             :require => false
+gem "manageiq-password",                "~>0.3",             :require => false
+gem "manageiq-postgres_ha_admin",       "~>3.1",             :require => false
+gem "manageiq-ssh-util",                "~>0.1.1",           :require => false
+gem "memoist",                          "~>0.16.0",          :require => false
+gem "money",                            "~>6.13.5",          :require => false
+gem "more_core_extensions"                                                     # min version should be set in manageiq-gems-pending, not here
+gem "net-ldap",                         "~>0.16.1",          :require => false
+gem "net-ping",                         "~>1.7.4",           :require => false
+gem "openscap",                         "~>0.4.8",           :require => false
+gem "optimist",                         "~>3.0",             :require => false
+gem "pg",                                                    :require => false
+gem "pg-dsn_parser",                    "~>0.1.0",           :require => false
+gem "psych",                            "~>3.1",             :require => false # This can be dropped once we drop ruby 2.5
+gem "query_relation",                   "~>0.1.0",           :require => false
+gem "rails",                            "~>6.0.0"
+gem "rails-i18n",                       "~>6.x"
+gem "rake",                             ">=12.3.3",          :require => false
+gem "rest-client",                      "~>2.1.0",           :require => false
+gem "ripper_ruby_parser",               "~>1.5.1",           :require => false
+gem "ruby-progressbar",                 "~>1.7.0",           :require => false
+gem "rubyzip",                          "~>2.0.0",           :require => false
+gem "rugged",                           "~>1.1",             :require => false
+gem "snmp",                             "~>1.2.0",           :require => false
+gem "sprockets",                        "~>3.7.2",           :require => false
+gem "sqlite3",                          "~>1.4.0",           :require => false
+gem "sync",                             "~>0.5",             :require => false
+gem "sys-filesystem",                   "~>1.3.4"
+gem "terminal",                                              :require => false
 
 # Custom gem that replaces mime-types in order to redirect mime-types calls to mini_mime
 #   Source is located at https://github.com/ManageIQ/mime-types-redirector
-gem "mime-types",                     "~>3.0",     :source => "https://rubygems.manageiq.org", :require => false
+gem "mime-types",                       "~>3.0",             :require => false, :source => "https://rubygems.manageiq.org"
 
 # Modified gems (forked on Github)
-gem "ruport",                         "=1.7.0.3",  :source => "https://rubygems.manageiq.org"
+gem "handsoap",                         "=0.2.5.5",          :require => false, :source => "https://rubygems.manageiq.org"
```

Comment:
Can we add a comment here that it's only because of manageiq-gems-pending? (that's why we had it up near mangaeiq-gems-pending)